search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityService.kt

```diff
@@ -7,10 +7,18 @@ import org.springframework.stereotype.Service
 import java.net.URI
 
 typealias SimplifiedTemporalAttribute = Map<String, Any>
+typealias QueryTemporalEntitiesResult = List<Pair<URI, Map<TemporalEntityAttribute, List<AttributeInstanceResult>>>>
 
 @Service
 class TemporalEntityService {
 
+    fun buildTemporalEntities(
+        queryResult: QueryTemporalEntitiesResult,
+        temporalQuery: TemporalQuery,
+        contexts: List<String>,
+        withTemporalValues: Boolean
+    ) = queryResult.map { buildTemporalEntity(it.first, it.second, temporalQuery, contexts, withTemporalValues) }
```

Comment:
add explicitely the return type of the function (harder to read either)

Reply:
Done